packages/@ember/polyfills/lib/assign.ts

```diff
@@ -7,7 +7,7 @@ import { deprecate } from '@ember/debug';
 export function assign<T, U>(target: T, source: U): T & U;
 export function assign<T, U, V>(target: T, source1: U, source2: V): T & U & V;
 export function assign<T, U, V, W>(target: T, source1: U, source2: V, source3: W): T & U & V & W;
-export function assign(target: object, ...sources: object[]): object;
+export function assign(target: object, ...sources: any[]): any;
```

Comment:
:sigh: This is probably the “right” thing to do here, since it’s what TS itself has, but I hate it.